This patch fixes issues with the report not working
when the SCE script output contains special characters.

will be submitted to upstream
--- openscap-1.3.7/src/common/oscap_acquire.c
+++ openscap-1.3.7/src/common/oscap_acquire.c
@@ -436,6 +436,15 @@ if (readbuf == '&') {
 			// & is a special case, we have to "escape" it manually
 			// (all else will eventually get handled by libxml)
 			oscap_string_append_string(pipe_string, "&amp;");
+                } else if (readbuf < 0x20 && !(readbuf == '\t' ||
+               readbuf == '\n' || readbuf == '\r')) {
+            // libxml doesn't tolerate most control characters
+            // in its input stream, reformat
+            char tranbuf[5];
+
+            snprintf(tranbuf, sizeof(tranbuf), "\\%03o", readbuf);
+            oscap_string_append_string(pipe_string, tranbuf);
+
 		} else {
 			oscap_string_append_char(pipe_string, readbuf);
 		}
